pythonFiles/vscode_datascience_helpers/kernel_launcher.py

```diff
@@ -11,9 +11,6 @@
 # See comment at the point of our use of Popen
 from subprocess import Popen, PIPE  # nosec
 
-from ipython_genutils.encoding import getdefaultencoding
-from ipython_genutils.py3compat import cast_bytes_py2
```

Comment:
This is no longer required, was only required for python 2.7
Also ipython_genutils is no longer shipping in conda